src/server/game/Entities/Player/Player.cpp

```diff
@@ -28867,3 +28867,34 @@ uint8 Player::GetItemLimitCategoryQuantity(ItemLimitCategoryEntry const* limitEn
 
     return limit;
 }
+
+Stats Player::GetPrimaryStats() const
+{
+    switch (getClass())
```

Comment:
I will provide a better implementation for this function that does not hardcode classes and specializations